aries-site/src/examples/components/namevaluelist/data.js

```diff
@@ -0,0 +1,99 @@
+import { Anchor, Box } from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+export const alignmentData = {
+  'Created by': (
+    <Anchor label="jane.doe@email.com" href="mailto:jane.doe@email.com" />
+  ),
+  'Created on': '13/02/2021 at 14:23:32',
+  Policies: '13',
+};
+
+export const anatomyData = {
+  'Single Line Description': 'This is an example of a single-line description',
+  'Multi-line Description': `This is an example of a multi-line description like
+   full-service AI insights, security and unified infrastructure management for
+   campus, branch, remote, and data center networks`,
+  Anchors: (
+    <Anchor label="jane.doe@email.com" href="mailto:jane.doe@email.com" />
```

Comment:
Strange but doesn't the anchor look bigger than the rest of the text?

<img width="693" alt="Screen Shot 2021-11-02 at 11 28 39 AM" src="https://user-images.githubusercontent.com/12522275/139923827-6e99f7c6-71aa-4c52-a482-5664cc23b31a.png">

Reply:
so strange Ill look into this